### PR TITLE
All registered users should be able to get membership list of the space #45

### PR DESCRIPTION
**Pull request by gaurav-handysolver**
Now, you will get the list of members present in the space

Note: It will not check whether the user is a member of that space.

#45

### CODE REVIEW

File: docs/postman/Smart Village API.postman_collection.json [hunk 1]
```diff
@@ -307,6 +308,27 @@
 						}
 					},
 					"response": []
+				},
+				{
+					"name": "Membership - List",
+					"request": {
+						"method": "GET",
+						"header": [],
```
Comment (donni106) [inline docs/postman/Smart Village API.postman_collection.json:316]: is the bearer token missing? only registered users should be able to make the request. it seems to be missing here like for the other endpoint mentioned in #46.

Comment (gaurav-handysolver) [reply docs/postman/Smart Village API.postman_collection.json:316]: Hi @donni106 
You can pass the bearer token in request and save it

Comment (donni106) [reply docs/postman/Smart Village API.postman_collection.json:316]: But does the endpoint return an error without bearer token? For example if guests without login would try to fetch.

Comment (donni106) [reply docs/postman/Smart Village API.postman_collection.json:316]: I have tested it and it works well without bearer token:

```
{
  "name": "Unauthorized",
  "message": "Invalid token!",
  "code": 0,
  "status": "401",
  "type": "yii\\web\\HttpException"
}
```

**Pull request merged**